### ISSUE TITLE
Plugin fails to load

### ISSUE THREAD

**Issue by stmach** [tarioch/beancounttools#16]
Hi!

I'm just starting out with beancount and have found your useful collection of tools. When adding your plugin to my beancount file (`plugin "tariochbctools.plugins.generate_base_ccy_prices" "CHF"`) beancount spits out the following error:
```
<load>:0:       Error importing "tariochbctools.plugins.generate_base_ccy_prices": unsupported operand type(s) for *: 'decimal.Decimal' and 'NoneType'`
```
I'm using `tariochbctools` v0.6.0, `beancount` v2.2.3 and Python v3.7.4 on Windows.

**Comment by tarioch** [comment]
Interesting, I'm using it only on linux but I don't see how that can make a difference.
I'm also using 3.7.6 but I guess that should also not be the problem. Are you using it from native windows or wsl?
Could you try it out from wsl?

**Comment by stmach** [comment]
All right, I've spun up WSL on my machine (Ubuntu 20.04 LTS) using Python v3.8.2 and I still got the same error.

Upon investigating more, I found out that the error is related to having `price` entries in the file without resolving all of the RHS commodities to the base currency (the arguemnt provided to the plugin). For example:
```
plugin "tariochbctools.plugins.generate_base_ccy_prices" "BASE"

2020-05-09 price WHATEVER    77 SOMETHING
```
fails with the error. adding to the file:
```
2020-05-09 price SOMETHING   12 BASE
```
makes it go away. So I'm pretty sure a missing price for the base currency is the culprit.

In my case, I had `price`s for some stocks in USD, but was still missing a `price` entry for USD -> CHF (which I wanted to fetch in using your `exchangeratesapi`).

---
Another thing I see (and probably I should split this into its own issue):
When running `bean-price` for the first time (or with `--no-cache`) using `exchangeratesapi`, the name of the commodity is printed out as well (which will lead to issues if you want to `include` the resulting output):
Inside the beancount file:
```
1792-04-02 commodity USD
  name: "United States Dollar"
  assets-class: "cash"
  price: "CHF:tariochbctools.plugins.prices.exchangeratesapi/USD"
```
Then:
```
$ bean-price main.bean --no-cache
USD
2020-05-08 price USD                                  0.97 CHF
```

**Comment by stmach** [comment]
Addendum:
if there's already a `price` entry (for the base currency) specified for the current date it also fails.
So it fails if:
- `price` entries are present in the file but one going to base currency is not there at all
- `price` entry going to base currency for the current date is already in the file

**Comment by tarioch** [comment]
Thank you very much for the detailed tracking of the issues. All of them should be fixed in 0.6.1 that I just released. I now also added some test cases for the scenarios you described.
